src/os/bluestore/BlueStore.cc

```diff
@@ -7445,6 +7454,11 @@ void BlueStore::_txc_update_store_statfs(TransContext *txc)
   logger->inc(l_bluestore_compressed_allocated, txc->statfs_delta.compressed_allocated());
   logger->inc(l_bluestore_compressed_original, txc->statfs_delta.compressed_original());
 
+  {
+    std::lock_guard<std::mutex> l(vstatfs_lock);
```

Comment:
@ifed01 did you do benchmark runs with this lock introduction? Mostly will be better than reading from DB, did we observe any improvement?